content/en/docs/migration/ot_overview.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Migrating from OpenTelemetry 
+weight: 20
+---
```

Comment:
Could you also mark this page as draft for now. That will allow us to merge it without having it published to the website yet. I'll like to make a pass for copyedits. 

If you agree, then add `draft: true` to the front matter.